micall/tests/test_consensus_aligner.py

```diff
@@ -834,13 +858,23 @@ def test_count_coord_concordance_short_match():
     # last match position:             ^
     aligner.coordinate_name = 'test-region'
     aligner.alignments = [AlignmentWrapper(r_st=0, r_en=15, q_st=0, q_en=15, cigar=[[15, CigarActions.MATCH]])]
-    # as the averaging window (size 20) slides along the query, the concordance decreases from 15/20 = 0.75
-    # in 1 base intervals (1/20 = 0.05) down to 8 bases of the match left in the window (8/20 = 0.4)
-    expected_concordance_list = [None]*10 + [0.75, 0.7, 0.65, 0.6, 0.55, 0.5, 0.45, 0.4] + [None]*9
+    # We start out with 100% match for the first 6 positions
```

Comment:
I think it's helpful to mention that you have a 15 base alignment on a 27 base consensus. That way, someone reading this doesn't have to do the counting.

Reply:
Good idea, I added docstrings to all the concordance tests with those details.